src/java_core/lesson6/homework/Animal.java

```diff
@@ -0,0 +1,69 @@
+package java_core.lesson6.homework;
```

Comment:
![Changes for revision 65a4ff2e81cd42b31a72a30c4d532b74965d33e8_008](https://user-images.githubusercontent.com/36124067/78024489-4475da00-737a-11ea-841e-3dbb1c5a28ca.png)